### PR TITLE
chore: Additional changes for camelCase refactoring

### PR DESCRIPTION
**Pull request by gregor**

### CODE REVIEW

File: app/script/calling/CallingRepository.js [hunk 1]
```diff
@@ -903,15 +903,26 @@ z.calling.CallingRepository = class CallingRepository {
...
    *
-   * @param {string} conversationId - ID of conversation for which the user should be removed from the call
+   * @param {z.calling.entities.CallEntity} callEntity - Call entity
    * @param {string} userId - ID of user to be removed
    * @returns {undefined} No return value
    */
-  participantLeft(conversationId, userId) {
-    const additionalPayload = z.calling.CallMessageBuilder.createPayload(conversationId, this.selfUserId(), userId);
-    const callMessageEntity = z.calling.CallMessageBuilder.buildGroupLeave(false, this.sessionId, additionalPayload);
+  participantLeft(callEntity, userId) {
```
Comment (HemangiS) [inline app/script/calling/CallingRepository.js]: @gregor  Shouldn't it be `removeParticipant` here?

File: app/script/calling/CallMessageMapper.js [hunk 2]
```diff
@@ -22,55 +22,45 @@
...
   /**
    * Map incoming call message into entity.
    *
    * @private
    * @param {Object} event - Call event object
-   * @returns {CallMessage} Call message entity
+   * @returns {z.calling.entities.CallMessageEntity} Call message entity
    */
-  const _mapEvent = function(event) {
+  mapEvent(event) {
```
Comment (HemangiS) [inline app/script/calling/CallMessageMapper.js:33]: private function

File: app/script/calling/CallingRepository.js (hunk 1 above)
Comment (gregor) [reply app/script/calling/CallingRepository.js]: Nice catch. You are absolutely right.

File: app/script/calling/CallMessageMapper.js (hunk 2 above)
Comment (gregor) [reply app/script/calling/CallMessageMapper.js:33]: Not any longer. The self invoking function was definitely overengineered in this case. I am returning a simple object with the function directly exposed publicly.

**Comment by gregor** [comment]
Thanks for the review. I fixed the issue you found and this should be good to go now.

File: app/script/calling/CallMessageMapper.js [hunk 3]
```diff
@@ -22,55 +22,45 @@
...
+        const {
+          dest_clientid: destinationClientId,
+          dest_user_id: destinationUserId,
           props: properties,
-          sdp: sdp,
-        };
+          sdp,
+        } = callMessage;
+
+        content = {destinationClientId, destinationUserId, properties, sdp};
```
Comment (HemangiS) [inline app/script/calling/CallMessageMapper.js:49]: May be something wrong here. I can't understand.

Comment (gregor) [reply app/script/calling/CallMessageMapper.js:49]: No, it is all good. I destructure the format agreed between clients, user proper and explicit camelCase naming in our internal entities and ensure spec compliance in the `toJson` of the `CallMessageEntity`.

**Pull request merged**